data.sql

```diff
@@ -50,4 +50,64 @@ ROLLBACK jan_2022;
 
 /* Update all animals' weights that are negative to be their weight multiplied by -1. */
```

Comment:
##### I am making this a required change because I won't be able to populate data if this error is not fixed.
- [ ] In your data.sql file, the Date/Time field format for the `date_of_birth` column in animals tables are out of reach. Please, this is the piece of code I am referring to from **_lines 3_**:

![image](https://user-images.githubusercontent.com/77106962/173127778-f999103a-0edc-4aa4-89d2-d2a3066bc11b.png)

When I run the command for populating data into the tables, I get this error:

![image](https://user-images.githubusercontent.com/77106962/173127948-01c181ba-eb28-4afe-b228-608aa5b65949.png)

#### **_Hint:_** I would advise you to use the `year-month-day` format to populate the entries for the `date_of_birth` column. 👍